pkg/dao/repository_test.go

```diff
@@ -216,15 +216,31 @@ func (suite *RepositorySuite) TestBulkCreateOneFails() {
 	rr, err := GetRepositoryDao(tx).BulkCreate(requests)
 
 	assert.Error(t, err)
-	assert.Equal(t, 2, len(rr))
+	assert.Equal(t, len(requests), len(rr))
+	assert.NotNil(t, rr[0].ErrorMsg)
+	assert.Nil(t, rr[0].Repository)
+	assert.Nil(t, rr[1].ErrorMsg)
+	assert.Nil(t, rr[1].Repository)
 
 	daoError, ok := err.(*Error)
 	assert.True(t, ok)
 	assert.True(t, daoError.BadValidation)
 
-	foundRepoConfig := models.RepositoryConfiguration{}
-	err = tx.First(&foundRepoConfig).Error
-	assert.Error(t, err)
+	urls := []string{}
+	for _, request := range requests {
+		if request.URL != nil && *request.URL != "" {
+			urls = append(urls, *request.URL)
+		}
+	}
+	var count int64
+	foundRepoConfig := []models.RepositoryConfiguration{}
+	err = tx.Model(&models.RepositoryConfiguration{}).
+		Where("repositories.url in (?)", urls).
```

Comment:
We may want to also filter by OrgID here and anywhere else that isn't querying by UUID